platform/src/main/resources/config/anchor-config-default-values.yaml

```diff
@@ -145,27 +145,43 @@ event_processor:
   callback_api_request:
     enabled: true
 
-# The list of clients for the Anchor server to safely communicate with the outside client domains.
+# The list of clients for the Anchor server to safely communicate with the outside wallet servers or clients.
 # Each item in the list may contain the following fields:
-#   - domain: the domain of the client
-#   - type: the type of the client. The supported values are `custodial` and `non-custodial`
-#   - public_key: the public key of the client. This must match the public key of the TOML file of the client domain
-#   - url: the URL of the client's callback API endpoint
+#   - type: `custodial` or `noncustodial`
+#
+#   If the type is `custodial`,
+#   - account: (required) the custodial account of the client. If the `account` of the transaction is the same as this
```

Comment:
How is this field going to be used? Is it for whitelisting SEP-10/SEP-24 calls?
If yes, I think we need to have a list of accounts, as wallets may use different account for withdrawal/deposit/authentication

Reply:
@Ifropc @JakeUrban Can a custodial wallet have multiple accounts?

Reply:
They will likely only use one account to authenticate, but also likely will have many accounts for sending & receiving payments.

Reply:
In this case, should we change `account` to `accounts` to allow multiple accounts?

Reply:
I would use this scheme:

```
- type: noncustodial
  domain: lobstr.co
  callback_url: https://lobstr.co/callback
  signing_key: G...  # optional
- type: custodial
  public_key: G...
  callback_url: https://whitebit.com/callback
```

In short:
- use signing_key instead of public_key for noncustodial wallets to avoid name overloading
- using public_key instead of account for custodial wallets because the stellar account doesn't need to exist for SEP-10 auth
- eventually, we may want to add `destination_accounts` and `source_accounts` lists to the custodial client config, but we don't need to do that until requested

Reply:
+1 to Jake

Reply:
We will do ``destination_accounts` and `source_accounts` when they are requested.

Reply:
Yeah I think we should separate auth_key from other types of keys